src/parser.rs

```diff
@@ -867,6 +1036,38 @@ mod tests {
     }
 
     #[test]
+    fn test_nullary_test() {
+        let mut parser = Rdp::new(StringInput::new("is defined"));
+        assert!(parser.test());
+        assert!(parser.end());
+    }
+
+    #[test]
+    fn test_unary_test() {
+        let mut parser = Rdp::new(StringInput::new("is equalto other"));
+        assert!(parser.test());
+        assert!(parser.end());
+
+        let mut parser = Rdp::new(StringInput::new("is equalto(other)"));
```

Comment:
This is what Jinja2 does. It's pretty neat to be able to write "my_var is odd" instead of "my_var is odd()".